cmd/kube-apiserver/app/server.go

```diff
@@ -460,32 +461,30 @@ func (s *APIServer) Run(_ []string) error {
 		glog.Fatalf("Invalid server address: %v", err)
 	}
 
-	legacyV1Group, err := latest.Group("")
-	if err != nil {
-		return err
+	if !latest.IsEnabled() {
```

Comment:
For this part of the code, I want to refactor it to a loop over all groups registered in the latest package. This will only be possible if we use the centralized latest.